src/com/facebook/buck/testrunner/TestNGRunner.java

```diff
@@ -55,9 +63,18 @@ public void run() throws Throwable {
       } else {
         results = new ArrayList<>();
         TestNG testng = new TestNG();
+        testng.setUseDefaultListeners(false);
         testng.setAnnotationTransformer(new FilteringAnnotationTransformer(results));
-        testng.setTestClasses(new Class<?>[] {testClass});
```

Comment:
looks like you're breaking formatting here... Please use google-format to fix all formatting issues

Reply:
I installed the IntelliJ plugin, but I guess it doesn't auto-format :(